app/helpers/proteus/helpers.rb

```diff
@@ -27,7 +27,7 @@ def proteus_stylesheet_link_tag(name, opts={})
       # probably hook into the compile_assets_fallback option or something
 
       puts "Checking for domain #{request.host.downcase.strip}"
-      domain = WhitelabeledDomain.find_by(host: request.host.downcase.strip)
+      domain = ::Proteus::WhitelabeledDomain.find_by(host: request.host.downcase.strip)
```

Comment:
We might need to audit further for more of these... If this clears the error I'll try to update it everywhere.

Reply:
Yeah makes sense. So far so good but I guess we'll see just how well it works in the next couple of days